facebook/draft-js
# removes snoflwake tex linting, and lints examples

**Summary**

this removes the special tex example linting. that had it's own
copy of eslint linting just itself. after removing eslint i removed
dependencies i didn't see were being used. after that it failed to
build, not reconizing the jsx/import statements. so i added the two
babel presets to handle that. (react, and es2015 presets).

next this starts linting the `examples/` directory. this should
introduce no new warnings/errors. as only the tex example had two
things that didn't comply (trailing commas, which have been fixed).

**Test Plan**

1. Install the project.
2. Run `npm run lint`, and ensure it passes.
3. Go to the TeX example: `cd examples/tex`, and ensure it builds: `npm install && npm start`

fixes #536
